### ISSUE TITLE
Importing the CSS file requires the package "tippy.js" in addition to "@tippyjs/react"

### ISSUE THREAD

**Issue by ellvtr** [atomiks/tippyjs-react#391]
Importing the CSS file requires the package "tippy.js" in addition to "@tippyjs/react".

This line from your documentation causes a compile error:
`import 'tippy.js/dist/tippy.css'; // optional`

Solution is to install  "tippy.js" in addition to "@tippyjs/react".

If that is on purpose, please include it in the documentation.
If not, what is the proper way to import the styles?

**Comment by yansena** [comment]
> Importing the CSS file requires the package "tippy.js" in addition to "@tippyjs/react".
> 
> This line from your documentation causes a compile error: `import 'tippy.js/dist/tippy.css'; // optional`
> 
> Solution is to install "tippy.js" in addition to "@tippyjs/react".
> 
> If that is on purpose, please include it in the documentation. If not, what is the proper way to import the styles?


Exactly, I’m trying to build a third project and the build crash because I can’t find the tippy.css file of tippy.js/dist because the main library is not installed, recommend include in the installation the central library also to stop this crash.

**Comment by ellvtr** [comment]
My workaround is to copy the file 'tippy.js/dist/tippy.css' from the old "tippy.js" package and remove that package again.
In most projects, you'd need to make custom CSS anyway. 
But the documentation should be fixed nevertheless. And I suggest including that css file in the new package "@tippyjs/react", so people have a default starting point.
